### DISCUSSION TITLE
Desafio para o Grupo

### DISCUSSION THREAD

**Discussion by HamiltonFilho** [ErickViniciusX/IT-BRAZIL-ACADEMY-SALTO-CC-GRUPO-05#20] [General]
Caro Grupo,

Vocês conseguem completar pelo menos uma história até a próxima aula, e demonstrar para a sala?

Encarem este desafio como pedido inesperado do cliente, o que acontece no mundo real e a equipe precisa se organizar

Mesmo que não seja possível, gostaria muito de entender todos os fatos e lições aprendidas. Por favor tomem notas!

**Comment by ErickViniciusX** [comment]
Olá Professor, estamos já com uma parte pronta, porem vamos continuar desenvolvendo.

**Comment by HamiltonFilho** [comment]
Ótimo!

A ideia é mostrar o que vcs já tem pronto! Que tal fazer uma demo amanhã na sala?

O mais importante são essas discussões, feedback e demos em torno de um produto para estimular a criatividade da equipe. Isso certamente alavancará ainda mais futuros desenvolvimentos
